src/components/GameView/CardListSortable.vue

```diff
@@ -84,15 +87,14 @@ export default {
   },
   computed: {
     sortedCards() {
-      return this.sortByRank ? sortBy(this.cards, 'rank') : [...this.cards];
+      return this.sortByRank ? orderBy(this.cards, ['rank', 'suit']) : [...this.cards]; // Changed to use orderBy
```

Comment:
We don't need the permanent comment that thus was changed
```suggestion
      return this.sortByRank ? orderBy(this.cards, ['rank', 'suit']) : [...this.cards];
```